eahub/templates/components/feedback.html

```diff
@@ -0,0 +1,144 @@
+{% block feedback %}
+
+<style>
+  @media (min-width: 1200px) {
+  #feedback {
+    right: -420px;
+    width: 420px;
+    }
+  }
+  
+  @media (max-width: 1200px) {
+  #feedback {
+    right: -200px;
+    width: 200px;
+  }
+
+  #closebtn {
+    display: none;
+  }
+  }
+
+  /*--- All styles ---*/
+
+  #feedback {
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    min-height: 450px;
+    position: fixed;
+    top: 50%;
+    transform: translateY(-50%);
+    transition: right 0.5s linear;
+    padding: 0 58px 58px;
+    z-index: 1090;
+  }
+
+  #feedback.active {
+      right: 0;
+      box-shadow: 0 10px 18px rgba(0,0,0,0.16);
+  }
+  
+  .modal-content {
+    height: 100%;
+  }
+  
+  #overlay {
+    background-color: rgba(0,0,0,0.4);
+      position: fixed;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      z-index: 1080;
+      opacity: 0;
+      visibility: hidden;
+      transition: all 0.5s linear;
+  }
+  
+  #overlay.active {
+    visibility: visible;
+    opacity: 1;
+  }
+  
+  #feedbackModal{
+    display: block;
+  }
+
+  .feedbackbtn {
+    border: 0;
+    color: #fff;
+    background: rgb(99, 82, 116);
+    border-color: rgb(99, 82, 116);
+    cursor: pointer;
+    line-height: 44px;
+    left: 0;
+    padding: 0 28px;
+    position: absolute;
+    text-align: center;
+    top: 50%;
+    transform: rotate(-90deg) translate(-50%, -100%);
+    transform-origin: top left;
+    box-shadow: 0 0 0 rgba(0,112,153,0.4);
+  }
+
+  #closebtn {
+    color: #fff;
+  }
+
+  .submitbtn {
+    color: #fff;
+    background: rgb(99, 82, 116);
+    border-color: rgb(99, 82, 116);
+  }
+</style>
+  
+  <div id="overlay"></div>
+
+  <div id="feedback">
+  <!-- Button trigger modal -->
+  <button type="button" class="btn feedbackbtn" onClick="togglemodal()">
+      Feedback?
+    </button>
+    
+    <!-- Modal -->
+    <div class="modal fade" id="feedbackModal" tabindex="-1">
+      <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title">Leave Feedback</h5>
+          </div>
+          <div class="modal-body">
+            <p>Do you have any feedback for us?</p>
+            <form class="form-group" method="post" action="{% url 'feedback:submit_feedback' %}">
+              {% csrf_token %}
+              <textarea class="form-control" name="message" cols="37" rows="5"></textarea>
```

Comment:
@LisaJD FYI you can add the `required` attribute here to enable the native browser validation, same with the <input> below 👍

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required

Reply:
I added this in the commit today but it doesn't seem to do anything?